tsl/src/nodes/decompress_chunk/compressed_batch.c

```diff
@@ -398,89 +520,13 @@ compute_plain_qual(DecompressContext *dcontext, DecompressBatchState *batch_stat
 	 * Find the compressed column referred to by the Var.
 	 */
 	Var *var = castNode(Var, linitial(args));
-	CompressionColumnDescription *column_description = NULL;
-	int column_index = 0;
-	for (; column_index < dcontext->num_data_columns; column_index++)
-	{
-		column_description = &dcontext->compressed_chunk_columns[column_index];
-		if (var->varno == INDEX_VAR)
-		{
-			/*
-			 * Reference into custom scan tlist, happens when we are using a
-			 * non-default custom scan tuple.
-			 */
-			if (column_description->custom_scan_attno == var->varattno)
-			{
-				break;
-			}
-		}
-		else
-		{
-			/*
-			 * Reference into uncompressed chunk tuple.
-			 *
-			 * Note that this is somewhat redundant, because this branch is
-			 * taken when we do not use a custom scan tuple, and in this case
-			 * the custom scan attno is the same as the uncompressed chunk attno,
-			 * so the above branch would do as well. This difference might
-			 * become relevant in the future, if we stop outputting the
```

Comment:
I assume that the code was just moved here. If that is not the case, please tell me and I'll review it.

Reply:
yes, this was moved, but I believe there are a few lines changes to use the new interfaces.